test/helpers/helper_functions.js

```diff
@@ -23,6 +25,47 @@ function readRequestQueue() {
     var a = JSON.parse(fs.readFileSync(reqDir, "utf-8")).cly_queue;
     return a;
 }
+function readBulkEventQueue() {
+    var a = JSON.parse(fs.readFileSync(bulkEventDir, "utf-8")).cly_bulk_event;
+    return a;
+}
+function readBulkReqQueue() {
+    var a = JSON.parse(fs.readFileSync(bulkQueueDir, "utf-8")).cly_req_queue;
+    return a;
+}
+
+function clearStorageJsonFiles(directory, files) {
```

Comment:
we already have clearstorage method below. you can modify it so that we dont remove things one by one. try using rmdir() instead for example and see what happens